src/app/shared/helpers/duration.ts

```diff
@@ -57,6 +57,10 @@ export class Duration {
     }
   }
 
+  getMS(): number {
```

Comment:
`getMs` instead? Otherwise we may think it is "get minutes and seconds".

Reply:
Fixed